lib/src/Uncertainty/Distribution/openturns/MarginalDistribution.hxx

```diff
@@ -67,7 +70,24 @@ public:
   void setIndices(const Indices & indices);
   Indices getIndices() const;
 
-private:
+  /** UsePDF accessor */
```

Comment:
This is an interesting new feature: thank you for this.

It creates, however, new undocumented methods:

![image](https://github.com/openturns/openturns/assets/31351465/a7512179-ed22-4781-839a-603415630ba6)